packages/core/src/components/TextField/TextField.scss

```diff
@@ -64,6 +68,14 @@ Style guide: components.text-field
   }
 }
 
+.ds-c-field--tiny {
+  max-width: 6em;
```

Comment:
Should we define these as variables instead, in case a team wants to override them? We could put them in `_variables.forms.scss`